.build/init.sh

```diff
@@ -0,0 +1,2 @@
+# Create a volume to reuse downloaded maven dependencies
```

Comment:
let add a default sha-bang for .sh scripts `#! /bin/bash`

Reply:
Added `#!/usr/bin/env bash` instead, looks like it's more portable